lib/ansible/modules/debug.py

```diff
@@ -28,7 +28,7 @@
     - A variable name to debug.
     - Mutually exclusive with the O(msg) option.
     - Be aware that this option already runs in Jinja2 context and has an implicit C({{ }}) wrapping,
-      so you should not be using Jinja2 delimiters unless you are looking for double interpolation.
+      so you should not be using Jinja2 delimiters unless you are looking for double interpretation.
```

Comment:
Interpolation is a good word to use here. https://en.wikipedia.org/wiki/String_interpolation